src/firebase.js

```diff
@@ -12,6 +12,16 @@ const firebaseConfig = {
   measurementId: "G-P1RQH1BG36",
 };
 
+// here i am using .env.local file to access firebase configuration;
+// const firebaseConfig = {
+//   apiKey: process.env.REACT_APP_FIREBASE_API_KEY,
+//   authDomain: process.env.REACT_APP_FIREBASE_AUTH_DOMAIN,
+//   projectId: process.env.REACT_APP_FIREBASE_PROJECT_ID,
+//   storageBucket: process.env.REACT_APP_FIREBASE_STORAGE_BUCKET,
+//   messagingSenderId: process.env.REACT_APP_FIREBASE_MESSAGING_SENDER_ID,
+//   appId: process.env.REACT_APP_FIREBASE_APP_ID,
+//   measurementId: process.env.REACT_APP_FIREBASE_MEASUREMENT_ID,
+// };
```

Comment:
What I meant was not to include the `'firebase config'` in the Pull request, you can use them on the local system.